packages/calcite-components/src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -641,22 +641,26 @@ describe("calcite-input-time-picker", () => {
 
     inputTimePicker.setProperty("lang", "da");
     await page.waitForChanges();
```

Comment:
Could you add a comment describing why we need to call `page.waitForChanges` in addition to waiting for a `requestAnimationFrame`?